docs/porting/overview.md

```diff
@@ -1,5 +1,82 @@
+- You don't have a need for more flexible deployment or self-contained console apps
+- You're happy with your code running on .NET Framework as it is today
+- Your code depends heavily on COM interop
+
+If it makes sense to port, then read on!
+
+## Overview of the Porting Process
+
+The recommended process for porting follows a series of steps.  Here's a short overview of the process in general.  Each of these parts of the process are covered in more detail in further articles.
+
+1. Identify and account for your third-party dependencies.
+
+   This will involve understanding what your third-party dependencies are, how you depend on them, how to see if they also run on .NET Core, and steps you can take if they don't.
+   
+2. Change all projects you wish to port to target .NET Framework 4.6.1.
+
+   This ensures that you can use API alternatives for .NET Framework-specific targets in the cases where .NET Core can't support a particular API.
+   
+3. Use the [API Portability Analyzer tool](tooling.md) to analyze your assemblies and develop a plan based on its results.
+
+   The API Portability Analyzer tool will analyze your compiled assemblies and generate a report which shows a high-level portability summary and a breakdown of each API you're using that isn't available on .NET Core.  You can use this report alongside an analysis of your codebase to develop a plan for how you'll port your code over.
+   
+4. Port your tests over to xUnit and NUnit.
+
+   MSTest doesn't support .NET Core, so you'll have to port your tests to either xUnit or NUnit if you wish to test your .NET Core targets.  Because porting to .NET Core is such a big change to your codebase, it's highly recommended to get your tests ported so that you can run tests as you port code over.
+   
+5. Determine which project system in Visual Studio is appropriate for your needs.
+
+   There are now two project systems in Visual Studio which support .NET Core: the traditional project system and the new `xproj` project system.  Both offer capabilities that the other does not, so it's critical that you pick the correct project system.  Details on this decision can be found in [Porting your libraries](libraries.md) and [Porting your applications](applications.md).
+   
+6. Execute your plan for porting!
+
+## Tools to help
+
+Here's a short list of the tools you'll likely use when porting to .NET Core:
+
+* NuGet - [client](https://dist.nuget.org/index.html) or [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer), the package manager for the .NET Platform.
+* ApiPort - [command line tool](https://github.com/Microsoft/dotnet-apiport/releases) or [Visual Studio Extension](https://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b), a toolchain that can generate a report of how portable your code is between .NET Framework and .NET Core, with an assembly-by-assembly breakdown of issues.
```

Comment:
Previously in this topic you called this "API Portability Analyzer tool". Should the names match in both sections? Should you link to the tooling topic for more information too?